### PR TITLE
more linting

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**

### CODE REVIEW

File: src/build_backend.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 TAG = 'py3-none-any'
 
 
-def find_sources(srcpkg: bool) -> Iterable[str]:
+def find_sources(*, srcpkg: bool) -> Iterable[str]:
```
Comment (martinpitt) [inline src/build_backend.py:17]: I wonder if this works on RHEL 8 with py 3.6 -- I faintly remember having to revert these at some point. The tests will tell us!

**Review by martinpitt — APPROVED** [review]
If this goes green, I'm happy with it, thanks! If you need to drop the `*` args, then please fix the commit message. Cheers!

File: src/cockpit/channels/filesystem.py [hunk 2]
```diff
@@ -120,10 +120,10 @@ def do_yield_data(self, options: Dict[str, object]) -> GeneratorChannel.DataGene
 
         except FileNotFoundError:
             return {'tag': '-'}
-        except PermissionError:
```
[the remark this answers is not in the document]
Comment (allisonkarlitskaya) [reply src/cockpit/channels/filesystem.py:123]: Yup.  I rewrote that commit message at some point and clearly missed a spot.

File: src/build_backend.py (hunk 1 above)
Comment (martinpitt) [reply src/build_backend.py:17]: [seems to be happy](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18830-20230522-124210-c6502549-centos-8-stream-pybridge/log.html) :+1:

**Comment by martinpitt** [comment]
I think these [testConnection](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18830-20230522-130434-c6502549-rhel4edge/log.html#14-2) failures are related to that whole saga of #18787, #18808, https://github.com/cockpit-project/cockpit/pull/18757#issuecomment-1543749060

**Pull request merged**